packages/special-pages/index.mjs

```diff
@@ -113,15 +118,26 @@ for (const copyJob of copyJobs) {
     }
 }
 for (const buildJob of buildJobs) {
-    if (DEBUG) console.log('BUILD:', relative(ROOT, buildJob.src), relative(ROOT, buildJob.dest))
+    if (DEBUG) console.log('BUILD:', buildJob.entryPoints, relative(ROOT, buildJob.outputDir))
     if (DEBUG) console.log('\t- import.meta.env: ', NODE_ENV)
     if (DEBUG) console.log('\t- import.meta.injectName: ', buildJob.injectName)
     if (!DRY_RUN) {
         buildSync({
-            entryPoints: [buildJob.src],
-            outfile: buildJob.dest,
+            entryPoints: buildJob.entryPoints,
+            outdir: buildJob.outputDir,
             bundle: true,
             format: 'iife',
+            // external: ['../assets/img/*'],
+            sourcemap: NODE_ENV === 'development',
+            // publicPath: '/js',
+            loader: {
+                '.js': 'jsx',
+                '.module.css': 'local-css',
+                '.svg': 'file',
+                '.jpg': 'file',
+                '.png': 'file',
+                '.riv': 'file'
```

Comment:
`jsx` -> for the preact components
`local-css` -> for the scoped css
`file` -> just uses the files as-is without processing